streamparse/storm/component.py

```diff
@@ -204,13 +205,13 @@ def _setup_component(self, storm_conf, context):
                                             1000000)  # 1 MB
             backup_count = self.storm_conf.get('streamparse.log.backup_count',
                                                10)
-            log_file = ('{log_path}/streamparse_{topology_name}_{component_name}'
-                        '_{task_id}_{pid}.log'
-                        .format(log_path=log_path,
-                                topology_name=self.topology_name,
-                                component_name=self.component_name,
-                                task_id=self.task_id,
-                                pid=self.pid))
+            log_file = join(log_path,
+                            ('streamparse_{topology_name}_{component_name}'
+                             '_{task_id}_{pid}.log'
+                             .format(topology_name=self.topology_name,
+                                     component_name=self.component_name,
+                                     task_id=self.task_id,
+                                     pid=self.pid)))
```

Comment:
Scratch that, I just remembered. If things like `invoke` run on a windows machine and create paths, they'll be `\` separated.  If it then deploys to Linux, everything explodes.  However, this is in the component, so clearly not a problem.